vignettes/pr_review_guidance.Rmd

```diff
@@ -21,107 +21,103 @@ knitr::opts_chunk$set(
-
+When a contributor opens a PR a lengthy standard text will be inserted into the comment section. Please do not alter any of the automated text. You will need to manually add `Closes #<insert_issue_number>` into the title of the Pull Request. You can use the Edit button in the top right if you forget this step at the start of your Pull Request. Besides that you are free to add in additional textual information, screenshots, etc. at the bottom of the automated text if needed to clarify or contribute to the discussion around your PR.
 
 ```{r echo=FALSE, out.width='120%'}
 knitr::include_graphics("./pr_review_checklist.png")
 ```
 
 ## Create a Pull Request
 
-After you click the green `Create pull request` button the automated text that was inserted will be turned into a checklist in your Pull Request. Each check box has been drawn from the previously mentioned vignettes and presented in the recommended sequence. These check boxes are meant to be a helpful aid in ensuring that you have created a compliant Pull Request. 
+After you click the green `Create pull request` button the automated text that was inserted will be turned into a checklist in your Pull Request. Each check box has been drawn from the previously mentioned vignettes and presented in the recommended sequence. These check boxes are meant to be a helpful aid in ensuring that you have created a compliant Pull Request.
 
 ```{r echo=FALSE, out.width='120%'}
 knitr::include_graphics("./pr_review_checkbox.png")
 ```
 
-## Complete the Pull Request checklist 
+## Complete the Pull Request checklist
 
 The check boxes are linked to the `task-list-completed` workflow. You need to check off each box in acknowledgment that you have done you due diligence in creating a compliant Pull Request. GitHub will refresh the Pull Request and trigger `task-list-completed` workflow that you have completed the task. The PR can not be merged into `devel` until the contributor has checked off each of the check box items.
 
 ```{r echo=FALSE, out.width='120%'}
 knitr::include_graphics("./pr_review_actions.png")
 ```
 
-Please don't hesitate to reach out to the `{admiral}` team on [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) or through the [GitHub Issues](https://github.com/pharmaverse/admiral/issues) tracker if you think this checklist needs to be amended or further clarity is needed on a check box item. 
+Please don't hesitate to reach out to the `{admiral}` team on [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) or through the [GitHub Issues](https://github.com/pharmaverse/admiral/issues) tracker if you think this checklist needs to be amended or further clarity is needed on a check box item.
 
 # GitHub Actions/Workflows
```

Comment:
So this might be crazy town.  But this is almost identical to the readme for admiralci.  Since that readme will be updated as we move forward, and we don't want to have outdated info here is there anyway we can snapshot that readme when we rebuild the site.   using an iframe or `include_graphics`.  

This could just be made into a separate issue as well.

Reply:
@bms63 crazy town is fun! I can create a separate issue and look into this.